package.json

```diff
@@ -17,9 +17,10 @@
     "storybook": "storybook dev -p ${STORYBOOK_PORT:-4479} -c storybook",
     "storybook:preview": "NODE_ENV=production pnpm storybook --docs",
     "storybook:clean": "rm -rf storybook/public",
-    "storybook:build:root": "storybook build -c storybook -o storybook/public",
-    "storybook:build:test:root": "pnpm storybook:build:root --test",
-    "storybook:build:prod:root": "pnpm storybook:build:root --docs",
+    "storybook:build": "storybook build -c storybook -o storybook/public",
+    "storybook:build:root": "pnpm turbo storybook:build",
```

Comment:
Partly wondered whether I should call it `storybook:build:full` or `:all` but that sounded pretty weird so I just kept it as `:root`. Open to clearer names if anyone can think of one.